scripts/generate_kmers.py

```diff
@@ -96,7 +96,8 @@ def find_kmers(pam, k, chrm, forward=True, end=True):
         if position < 0:
             continue
 
-        yield (kmer.upper(), position)
+        # Return the 1-indexed position to caller
+        yield kmer.upper(), position+1
```

Comment:
k-mers are assumed 1-indexed. Since this is a breaking change, I'm not sure if any files on the server need to be re-generated so they're not misinterpreted when we change `guidescan` version on the server. If so, perhaps we can open an issue on it.